src/Files.App/UserControls/Widgets/NetworkLocationsWidget.xaml

```diff
@@ -0,0 +1,146 @@
+	mc:Ignorable="d">
+
+	<Grid>
+		<!--  Message when there are no network locations  -->
+		<InfoBar
+			x:Name="NoNetworkLocationsInfo"
+			x:Load="{x:Bind ViewModel.IsNoNetworkLocations, Mode=OneWay}"
+			IsClosable="False"
+			IsIconVisible="True"
+			IsOpen="True"
+			Message="{helpers:ResourceString Name=NoNetworkLocationsInfo}"
+			Severity="Informational" />
+
+		<ItemsRepeater ItemsSource="{x:Bind ViewModel.Items, Mode=OneWay}">
+
+			<ItemsRepeater.Layout>
+				<UniformGridLayout
+					ItemsStretch="None"
+					MinColumnSpacing="8"
+					MinItemHeight="72"
+					MinItemWidth="240"
+					MinRowSpacing="8"
+					Orientation="Horizontal" />
+			</ItemsRepeater.Layout>
+
+			<ItemsRepeater.ItemTemplate>
+				<DataTemplate x:DataType="dataitems:WidgetDriveCardItem">
+					<Button
+						Padding="0"
+						HorizontalAlignment="Stretch"
+						VerticalAlignment="Stretch"
+						HorizontalContentAlignment="Stretch"
+						VerticalContentAlignment="Stretch"
+						AutomationProperties.Name="{x:Bind Item.Text, Mode=OneWay}"
+						Click="Button_Click"
+						CornerRadius="{StaticResource ControlCornerRadius}"
+						DataContext="{x:Bind}"
+						PointerPressed="Button_PointerPressed"
+						RightTapped="Button_RightTapped"
+						Tag="{x:Bind Item.Path}"
+						ToolTipService.ToolTip="{x:Bind Item.Text, Mode=OneWay}">
+						<Grid
+							Margin="8"
+							HorizontalAlignment="Stretch"
+							VerticalAlignment="Stretch"
+							ColumnSpacing="8">
+							<Grid.ColumnDefinitions>
+								<ColumnDefinition Width="32" />
+								<ColumnDefinition Width="*" />
+								<ColumnDefinition Width="Auto" />
+							</Grid.ColumnDefinitions>
+							<Grid.RowDefinitions>
+								<RowDefinition Height="*" />
+								<RowDefinition Height="*" />
+								<RowDefinition Height="*" />
+							</Grid.RowDefinitions>
+
+							<Image
+								Grid.RowSpan="3"
+								Grid.Column="0"
+								Width="32"
+								Height="32"
+								HorizontalAlignment="Center"
+								VerticalAlignment="Center"
+								x:Phase="1"
+								Source="{x:Bind Thumbnail, Mode=OneWay}"
+								Stretch="Uniform" />
+
+							<TextBlock
+								x:Name="ItemLocationName"
+								Grid.Row="0"
+								Grid.Column="1"
+								HorizontalAlignment="Stretch"
+								VerticalAlignment="Center"
+								FontSize="14"
+								FontWeight="Medium"
+								Text="{x:Bind Item.Text, Mode=OneWay}"
+								TextTrimming="CharacterEllipsis"
+								TextWrapping="NoWrap" />
+
+							<ProgressBar
+								x:Name="DriveSpaceProgressBar"
+								Grid.Row="1"
+								Grid.Column="1"
+								VerticalAlignment="Stretch"
+								VerticalContentAlignment="Stretch"
+								x:Load="{x:Bind Item.ShowDriveDetails, Mode=OneWay}"
+								AutomationProperties.AccessibilityView="Raw"
+								Maximum="{x:Bind Item.MaxSpace.GigaBytes, Mode=OneWay}"
+								Value="{x:Bind Item.SpaceUsed.GigaBytes, Mode=OneWay}" />
+
+							<Button
+								x:Name="GoToStorageSense"
```

Comment:
Do we need storage sense to network locations?

Reply:
Deleted.